From 7eb9c8bdb9020074765eb4ab08c31d79a91f2460 Mon Sep 17 00:00:00 2001
From: Alvaro Guzman <alvaro.guzman@owasys.com>
Date: Thu, 2 Mar 2023 12:27:31 +0100
Subject: [PATCH] <owa5x_config.OWA> Bridge netfilter

Added needed configs for it according usual docker configs
on linux kernel.

Changelog: added.
Signed-off-by: Alvaro Guzman <alvaro.guzman@owasys.com>
---
 owa5x_config.OWA | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/owa5x_config.OWA b/owa5x_config.OWA
index d21395e7b0d6..915b8d7de456 100644
--- a/owa5x_config.OWA
+++ b/owa5x_config.OWA
@@ -1006,13 +1006,14 @@ CONFIG_NET_PTP_CLASSIFY=y
 # CONFIG_NETWORK_PHY_TIMESTAMPING is not set
 CONFIG_NETFILTER=y
 CONFIG_NETFILTER_ADVANCED=y
-# CONFIG_BRIDGE_NETFILTER is not set
+CONFIG_BRIDGE_NETFILTER=m
 
 #
 # Core Netfilter Configuration
 #
 CONFIG_NETFILTER_INGRESS=y
 CONFIG_NETFILTER_NETLINK=m
+CONFIG_NETFILTER_FAMILY_BRIDGE=y
 # CONFIG_NETFILTER_NETLINK_ACCT is not set
 # CONFIG_NETFILTER_NETLINK_QUEUE is not set
 CONFIG_NETFILTER_NETLINK_LOG=m
@@ -1119,6 +1120,7 @@ CONFIG_NETFILTER_XT_MATCH_MULTIPORT=m
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
 CONFIG_NETFILTER_XT_MATCH_OWNER=m
 # CONFIG_NETFILTER_XT_MATCH_POLICY is not set
+# CONFIG_NETFILTER_XT_MATCH_PHYSDEV is not set
 CONFIG_NETFILTER_XT_MATCH_PKTTYPE=m
 CONFIG_NETFILTER_XT_MATCH_QUOTA=m
 # CONFIG_NETFILTER_XT_MATCH_RATEEST is not set
-- 
2.25.1

